lago/plugins/__init__.py

```diff
@@ -64,8 +65,8 @@ def load_plugins(namespace, instantiate=True):
     mgr = ExtensionManager(
         namespace=namespace,
         on_load_failure_callback=(
-            lambda _, ep, err: LOGGER.
-            debug('Could not load %r: %s', ep.name, err)
+            lambda _, ep, err: warnings.
+            warn('Could not load plugin {}: {}'.format(ep.name, err))
```

Comment:
I think using ``LOGGER.warning`` would be better here, iirc it would also go both to stderr and to the log file.
I wanted to use it also in the ``lago/__init__.py`` guestfs warning, but at that stage, logging is not setup yet.